### ISSUE TITLE
Latest VBA engine compatibility

### ISSUE THREAD

**Issue by cognociente** [govert/SQLiteForExcel#17]
Would this still work on the latest vba engine? I know there were some changes to how 64 bit vba needed to use pointers and call windows api functions...

**Comment by govert** [comment]
Yes, there is a version for 64-bit Excel too - https://github.com/govert/SQLiteForExcel/blob/master/Source/SQLite3VBAModules/Sqlite3_64.bas
